### PR TITLE
[idea] #453 Added language field to importsmethods index

### LINKED ISSUE #453: Add Language field to importsmethods index while producing tokens from Scala parser

### PR DESCRIPTION
**Pull request by keerath**
fixes #453
closes #455

### CODE REVIEW

**Comment by keerath** [comment]
@ScrapCodes please review..!!

File: core/src/main/scala/com/kodebeagle/indexer/JavaFileIndexer.scala [hunk 1]
```diff
@@ -32,6 +32,15 @@ object JavaFileIndexerHelper {
     s"""${repo.login}/${repo.name}/blob/${repo.defaultBranch}$actualFileName"""
   }
 
+  def getLang(fileName: String): String = {
+    val indexOfDot = fileName.indexOf('.')
```
Comment (ScrapCodes) [inline core/src/main/scala/com/kodebeagle/indexer/JavaFileIndexer.scala]: Are you sure about this, because there can be a dot in directory name as well.

Comment (keerath) [inline core/src/main/scala/com/kodebeagle/indexer/JavaFileIndexer.scala]: Well, i intended to write fileName.lastIndexOf... fixed it now..!!

**Comment by keerath** [comment]
@ScrapCodes Good to merge ..??

**Comment by ScrapCodes** [comment]
May be we could add some tests, otherwise it is good to go.

**Pull request merged**